### DISCUSSION TITLE
TabTabTab fonts are bold with Brave Browser and Windows 11?

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#374] [Q&A]
Hi,

With Windows 11 and Brave Browser: Fonts are bold
![Brave_Browser_Win11_TabTabTab](https://github.com/user-attachments/assets/48f79fb7-8ccd-49cb-b7ff-2aabad17786e)

With Windows 10 and Brave Browser: Fonts aren't bold
![Brave_Browser_Win10_TabTabTab](https://github.com/user-attachments/assets/e8de443d-f40b-4665-af08-669bd165f2c2)

With Microsoft Edge Browser both aren't bold.

?

**Comment by okaryo** [comment]
Hi,

The tab titles in TabTabTab use the `subtitle2` class from the Material UI library. This class styles the text with `fontWeight: 500` and `fontFamily: Roboto, Helvetica, Arial, sans-serif`. So, it's expected for the tab titles to appear bold. However, if they are not bold, it might be due to the browser not having the specific font installed or missing the fontWeight 500 version of the font.

If you could let me know which fonts are available on your browser, I can specify one of them in the `fontFamily` to ensure the text appears bold.

**Comment by FreePDF64** [comment]
Hi,

Thank you for your response. I do not want the fonts in bold.

Best Regards,
Michael

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by FreePDF64** [comment]
Would it be possible to change the font of the tab list as desired? I think this would definitely solve the font problem if you could choose a different font.

**Reply by okaryo** [reply]
Hi,

It is possible to add a font selection feature; however, this may unnecessarily complicate the application. If possible, I’d like to explore alternative ways to address the issue first. I’ll do some research to see if this can be resolved through library settings. Would it be okay if I get back to you after that?

Best regards,
Ryotaro

**Comment by FreePDF64** [comment]
That sounds very nice.

Thank you!

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by FreePDF64** [comment]
I may have a hint for you where to look:
In my Brave Browser the font looks like bold
![Note 1](https://github.com/user-attachments/assets/4dc53f17-c3bb-49e0-bc4a-c4c34292c51b)

but when I click on RESTORE the font looks like not bold
![Note 2](https://github.com/user-attachments/assets/f9b6bc44-6e5d-49b0-b35e-2483b12e11df)

The font should looks like always as you see in RESTORE.


I hope I could clarify a little bit this behaviour.

**Reply by okaryo** [reply]
Thanks! I'll check it.
